gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/StandardTableDefinition.java

```diff
@@ -252,12 +252,15 @@ ToStringHelper toStringHelper() {
   }
 
   @Override
-  public boolean equals(Object obj) {
-    return obj instanceof StandardTableDefinition && baseEquals((StandardTableDefinition) obj);
+  public final boolean equals(Object obj) {
```

Comment:
I am unsure whether we should make some of `TableDefinition` subclasses final (where your comment applies) and some other not (as this `StandardTableDefinition`) or just leave them as is for consistency in the hierarchy. Thoughts? /cc @aozarov